app/javascript/packs/building_map.js

```diff
@@ -120,3 +129,44 @@ function requestLocation(inputField){
         }
     );
 }
+
+const syncUserPositionWithServerImpl = async (location) => {
```

Comment:
I would suggest to put these functions in an extra .js package to give the project more structure.

Reply:
Why this one? It only applies to UI in the building map, what would the contents of the new file be?

Reply:
I also think this shouldn't be needed for the sync code, but the code for showing the position marker might be needed in other maps (as far as I'm aware, the user stories said there should be separate maps for shared positions).

Reply:
Hm, I tried to separate this out from the rest of this code, but it gets complicated rather quickly and opens up more questions that would block this PR even longer. I think we should just push this onto the list of follow-up tasks (see my initial work at 5fd064ca311ffaba20d39859f4d656a8206e6226)

Reply:
ok